### PR TITLE
tidy up

### PR DESCRIPTION
**Pull request by matthewtuby**
Restructured folders, removed test codes, and made changed minor code changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,15 @@
...
+## Installation
+
+Clone the repository!
+
 ```sh
-$ apt update
-$ apt install pip3
-$ pip3 install --user gym-retro
+$ cd reinforcement_learning/
+$ python3 -m pip install -r requirements.txt
```
Comment (qualname) [inline README.md:9]: Please commit your "requirements.txt" file too

File: example_codes/agent_vs_keyboard_example.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
+#
+# Controlls: MOVE: W A S D 
+#            HIT: right arrow
+#            BLOCK: left arrow
+#            KICK LOW: up arrow
```
Comment (qualname) [inline example_codes/agent_vs_keyboard_example.py]: Up and Down keys are possibly mixed up? See lines 47 and 48

File: example_codes/controller_vs_agent.py [hunk 3]
```diff
@@ -96,18 +113,19 @@ def act(self, observation, reward, done):
...
+ob, reward, done, info = env.step( action + "".join(pushed_keys) )
+while(True):
     for i in range(100000):
         pushed_keys = get_keys(pushed_keys)
         action = agent.act(ob, reward, done)
-        ob, reward, done, _ = env.step("".join(pushed_keys) + action )
+        ob, reward, done, info = env.step("".join(pushed_keys) + action )
         env.render()
         sleep(1/90)
+        if(info['wins'] == 1 or info['enemy_rounds_won'] == 1 or info['rounds_won'] == 1 or done or i > 4000):
```
Comment (qualname) [inline example_codes/controller_vs_agent.py:128]: This should be calculated by the environment (by changing how `done` is set) but that's just my opinion
(Also parentheses are so... javascript :D)

**Review by qualname — CHANGES_REQUESTED** [review]
Please fix the first 2 comments. Thanks!

File: example_codes/agent_vs_keyboard_example.py (hunk 2 above)
Comment (matthewtuby) [reply example_codes/agent_vs_keyboard_example.py]: They were, thanks for mentioning.

File: README.md (hunk 1 above)
Comment (matthewtuby) [reply README.md:9]: Committed!

File: example_codes/controller_vs_agent.py (hunk 3 above)
Comment (matthewtuby) [reply example_codes/controller_vs_agent.py:128]: For now it does the job :D,

**Review by qualname — APPROVED** [review]

**Pull request merged**